create_jenkins_job.py

```diff
@@ -296,6 +296,112 @@ def create_job(os_name, job_name, template_file, additional_dict):
             })
 
         # configure a manually triggered version of the coverage job
+
+        # Proposed list of packages to maximize coverage while testing quality level
+        # packages. The list is composed by the list of qualitly level packages plus
+        # packages of ros2.repos that are used by the qualitly level packages during
+        # tests.
+
+        # out of the list since ignored by colcon: shape_msgs, stereo_msgs, rmw_connext, rmw_cyclonedds
+        quality_level_pkgs = ['action_msgs',
```

Comment:
Nitpick: according to the style guide please don't align wrapped lines. Instead wrap after the `[` and indent the following just one level.

Same below.

Reply:
ouch sorry about that, fixed in d9a9997